src/crypto/public_key/elliptic/ecdsa.js

```diff
@@ -18,52 +18,43 @@
 // Implementation of ECDSA following RFC6637 for Openpgpjs
 
 /**
+ * @requires util
  * @requires crypto/hash
- * @requires crypto/public_key/jsbn
  * @requires crypto/public_key/elliptic/curves
  * @module crypto/public_key/elliptic/ecdsa
  */
 
+import util from '../../../util';
 import hash from '../../hash';
-import curves from './curves';
-import BigInteger from '../jsbn';
+import { get as curvesGet } from './curves';
 
 /**
  * Sign a message using the provided key
- * @param  {String}      oid        Elliptic curve for the key
- * @param  {enums.hash}  hash_algo  Hash algorithm used to sign
- * @param  {Uint8Array}  m          Message to sign
- * @param  {BigInteger}  d          Private key used to sign
- * @return {{r: BigInteger, s: BigInteger}}  Signature of the message
+ * @param  {module:type/oid} oid        Elliptic curve object identifier
+ * @param  {enums.hash}      hash_algo  Hash algorithm used to sign
+ * @param  {Uint8Array}      m          Message to sign
+ * @param  {Uint8Array}      d          Private key used to sign the message
```

Comment:
Why is this Uint8Array and it is BN for EdDSA?

Reply:
There are a few tests in `test/crypto/elliptic.js` that pass in arrays or typed arrays directly to this function. I can just as easily change those.

Reply:
Consistency is a virtue--I would change it if it's easy.